rendermodules/mesh_lens_correction/do_mesh_lens_correction.py

```diff
@@ -159,8 +211,14 @@ def run(self):
 
         args_for_input = dict(self.args)
 
+        self.maskUrl = make_mask(args_for_input)
+        # argshema doesn't like the NumpyArray after processing it once
+        # we don't need it after mask creation
+        self.args['mask_coords'] = None
+        args_for_input['mask_coords'] = None
+
         # create a stack with the lens correction tiles
-        ts_example = self.generate_ts_example()
+        ts_example = self.generate_ts_example(self.maskUrl)
```

Comment:
is it necessary to have this as an argument if it is already an attribute of `self`?